src/widgets/backuplistwidgetitem.cpp

```diff
@@ -91,7 +99,7 @@ void BackupListWidgetItem::updateDirDetail(quint64 size, quint64 count)
 {
     _size  = size;
     _count = count;
-    _ui.detailLabel->setText(QString::number(_count) + tr(" items totalling ")
+    _ui.detailLabel->setText(QString::number(_count) + tr(" items, ")
```

Comment:
Do you think we could use "files"? I've been thinking about this while doing the translation. Being a Unix head, to me it is obvious that files includes directories, but it might not be so for everyone. Items on the other hand is too generic.